Open Judge System/Services/OJS.Services/Business/ParticipantScores/IParticipantScoresBusinessService.cs

```diff
@@ -1,4 +1,8 @@
-﻿namespace OJS.Services.Business.ParticipantScores
+﻿using System.Collections.Generic;
+using OJS.Data.Models;
```

Comment:
move usings in a namespace or move the usings below which are in the namespace of it to be consistent